src/services/services.ts

```diff
@@ -1,10 +1,12 @@
+import {
+    SymbolObject,
+} from "../compiler/objectConstructors";
+import { SignatureObjectInternals } from "../compiler/signatureObjectInternals";
+import { SymbolObjectInternals } from "../compiler/symbolObjectInternals";

```

Comment:
Should we just do this via the namespaces? Half worried about the output ordering here but clearly it works so, it's probably fine. My other PR to sort imports more strictly is now moot because we dropped the plugin, so I can't enforce it anyway.